docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-cloud-providers/azure.md

```diff
@@ -212,14 +212,18 @@ roleRef:
 
 1. Select **External** from the **Cloud Provider** drop-down in the **Cluster Configuration** section.
 
+:::note
+When setting **Cloud Provider** to **External**, all nodes will automatically be tainted with node.cloudprovider.kubernetes.io/uninitialized=true until the Cloud Provder is installed. It is recommended to deploy the CPI as an add-on job as described below.
+:::
```

Comment:
```suggestion
  :::note

  When setting **Cloud Provider** to **External**, all nodes are automatically tainted with `node.cloudprovider.kubernetes.io/uninitialized=true` until the Cloud Provider is installed. It is recommended to deploy the CPI as an add-on job as described below.

  :::
```

Suggestion updates the text to present tense and adds indentation so the note aligns with the text of step 1.